var/spack/repos/builtin.mock/packages/openblas/package.py

```diff
@@ -17,6 +17,8 @@ class Openblas(Package):
     version("0.2.14", md5="b1190f3d3471685f17cfd1ec1d252ac9")
     version("0.2.13", md5="b1190f3d3471685f17cfd1ec1d252ac9")
 
+    depends_on("zlib")
+
```

Comment:
Let's setup a separate test, instead of modifying an existing package.

Reply:
I need a spec that has a 1st and 2nd level dependency that has the shared variant

Reply:
Sure, setup a different hierarchy just for that. Adding structure to existing packages already used in tests may influence the other tests too, so I'd avoid that.